test/test_linalg.py

```diff
@@ -3884,7 +3884,7 @@ def test(n=10,                       # how many tests to generate
                 args.append(out_sublist)
                 self._check_einsum(*args, np_args=(equation, *np_operands))
 
-        test(100)
+        test(1000)
```

Comment:
How slow is this test? Is it ok to bump this 10x?

Reply:
The CUDA version now goes to 3-4s, which isn't terrible, but let me tone this down to 500 instead, which should be sufficient. 
```
  test_einsum_random_cuda_complex128 (__main__.TestLinalgCUDA) ... ok (3.745s)
  test_einsum_random_cuda_float64 (__main__.TestLinalgCUDA) ... ok (3.368s)
```